tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1381,15 +1381,15 @@ describe("insertTestResult", () => {
 
     context("when inserting a testResult that has an LEC testType without a certificateNumber", () => {
         it("should throw 400 and descriptive error message", () => {
-            const testResultWithLecTestTypeWithoutCertNum = JSON.parse(JSON.stringify(testResultsPostMock[6]));
+            const testResultWithLecTestTypeWithoutCertNum = JSON.parse(JSON.stringify(testResultsPostMock[8]));
```

Comment:
You import lodash at the top of the file, but don't use it here.

Reply:
cloneDeep used